### ISSUE TITLE
Add pinball loss

### ISSUE THREAD

**Issue by RomainBrault** [tensorflow/addons#586]
**Describe the feature and the current behavior/state.**

The pinball loss is a very simple modification of the MAE and allows (conditional) quantile regression. This is rather simple to code, and would be of interest for a large community.

This has been submitted to TensorFlow (tensorflow/tensorflow#32871) and redirected here.

A pull request is open (#579)

**Relevant information**
- Are you willing to contribute it (yes/no): yes
- Are you willing to maintain it going forward? (yes/no): yes
- Is there a relevant academic paper? (if so, where): yes, [the original paper](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.470.9161&rep=rep1&type=pdf) and [many others](https://scholar.google.com/scholar?hl=fr&as_sdt=0%2C5&q=pinball+loss&btnG=)
- Is there already an implementation in another framework? (if so, where): yes, some blogs propose implementations (https://towardsdatascience.com/deep-quantile-regression-c85481548b5a)
- Was it part of tf.contrib? (if so, where): no

**Which API type would this fall under (layer, metric, optimizer, etc.)**

metrics and losses (Keras)

**Who will benefit with this feature?**

People in econmics, biology. To model data uncertainty / distribution.

**Any other info.**

**Comment by Sandy4321** [comment]
do you have  full example for multivariate quantile regressing with using
model = tf.keras.Model()
model.compile('sgd', loss=tfa.losses.PinballLoss(tau=.1))

from 
https://www.tensorflow.org/addons/api_docs/python/tfa/losses/PinballLoss
